internal/xds/translator/listener.go

```diff
@@ -39,6 +40,7 @@ func buildXdsTCPListener(name, address string, port uint32) *listenerv3.Listener
 				Filter:     listenerAccessLogFilter,
 			},
 		},
+		PerConnectionBufferLimitBytes: wrapperspb.UInt32(32768),
```

Comment:
should EG support to custom this?

Reply:
lets raise a issue for this ? #24 is always a back up option

Reply:
If so, we need to add new apis in envoyproxy too, let us keep it in this way, unless someone requests for it.